shell/platform/darwin/ios/platform_message_handler_ios.h

```diff
@@ -33,7 +32,7 @@ class PlatformMessageHandlerIos : public PlatformMessageHandler {
 
   struct HandlerInfo {
     NSObject<FlutterTaskQueueDispatch>* task_queue;
-    fml::ScopedBlock<FlutterBinaryMessageHandler> handler;
+    FlutterBinaryMessageHandler handler;
```

Comment:
Typedef is here:
https://github.com/flutter/engine/blob/eec1ba427f6a9cf54e44906837b8446172697fec/shell/platform/darwin/common/framework/Headers/FlutterBinaryMessenger.h#L30